include/cmm/Node.h

```diff
@@ -23,8 +23,9 @@ namespace cmm
     // TODO: Still needed??
```

Comment:
Though not a change related to this PR.  Let's just clean this up by removing it.

Reply:
Resolved in commit 5cdec0c